include/cling/Interpreter/Interpreter.h

```diff
@@ -102,6 +102,19 @@ namespace cling {
       ~StateDebuggerRAII();
     };
 
+    ///\brief Merge all transactions between construction and destruction
+    /// Use Prev to merge the Transactions with the last Transaction (true)
+    /// or the first Transaction created after instantiation (false).
+    class TransactionMerge {
```

Comment:
We traditionally use nouns for classes. Maybe `ScopedTransactionMerger`? (Well, we traditionally would have called that `TransactionMergerRAII`, choose whatever you prefer.)